### PR TITLE
Properly handle `artefact_extra_id`

### PR DESCRIPTION
**Pull request by 8R0WNI3**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator

```

### CODE REVIEW

**Review by zkdev — DISMISSED** [review]
/lgtm

File: src/components/dependencies/ComplianceCells.js [hunk 1]
```diff
@@ -250,7 +250,7 @@ const ComplianceCell = ({
       artefactSummary.artefact.artefact_kind === artefact.kind
       && artefactSummary.artefact.artefact.artefact_name === artefact.name
       && artefactSummary.artefact.artefact.artefact_version == artefact.version
-      && artefactSummary.artefact.artefact.artefact_type == artefact.type
+      && artefactSummary.artefact.artefact.artefact_type === artefact.type
```
Comment (zkdev) [inline src/components/dependencies/ComplianceCells.js:253]: Do we need to use strict equality operator here, or is it just a measure of precaution?

Comment (ccwienk) [reply src/components/dependencies/ComplianceCells.js:253]: @zkdev : I think it is a good practise to basically _always_ use strict equality-comparison, as the default one is severely broken (see the maaaany memes out there mocking ES for that ;-))

**Review by ccwienk — APPROVED** [review]

File: src/components/dependencies/ComplianceCells.js (hunk 1 above)
Comment (ccwienk) [reply src/components/dependencies/ComplianceCells.js:253]: image

Comment (8R0WNI3) [reply src/components/dependencies/ComplianceCells.js:253]: Actually, this was just some sort of cleanup and not directly related to the extra-id handling (in that: AFAIK, it did not lead to any bugs before).

**Pull request merged**